src/datasources/notifications/entities/notification-subscription.entity.db.ts

```diff
@@ -0,0 +1,104 @@
+import { NotificationDevice } from '@/datasources/notifications/entities/notification-devices.entity.db';
+import {
+  NotificationSubscriptionNotificationType,
+  NotificationSubscriptionNotificationTypeSchema,
+} from '@/datasources/notifications/entities/notification-subscription-notification-type.entity.db';
+import { AddressSchema } from '@/validation/entities/schemas/address.schema';
+import { NumericStringSchema } from '@/validation/entities/schemas/numeric-string.schema';
+import {
+  Column,
+  Entity,
+  Unique,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  OneToMany,
+  JoinColumn,
+} from 'typeorm';
+import { getAddress } from 'viem';
+import { z } from 'zod';
+
+export const NotificationSubscriptionSchema = z.object({
+  id: z.number(),
+  chain_id: NumericStringSchema,
+  safe_address: AddressSchema,
+  signer_address: AddressSchema.nullable(),
+  updated_at: z.date(),
```

Comment:
I think this is missing `created_at`. We can also extend the `RowSchema`.

Reply:
Done:
https://github.com/safe-global/safe-client-gateway/pull/2063/commits/5fc9e0bf5ba876172c32f7eb85554bf3e6b488ce

Reply:
Yes, I changed the other entities and missed this one. Done
https://github.com/safe-global/safe-client-gateway/pull/2063/commits/fd2a068ffd76d711dba2c3e7e24a5f7dee4b750d